From 0d9adc9caedfae096643b0ad7c5fefac22f7c8b0 Mon Sep 17 00:00:00 2001
From: Jakub Tyszkowski <jakub.tyszkowski@tieto.com>
Date: Tue, 16 Dec 2014 11:05:55 +0100
Subject: [PATCH 10/11] Revert "selinux: Add domain transition from logwrapper
 for bluetooth services"

We are not using logwrapper anymore.

This reverts commit f839d2022c9b73446c975f144144ca2cc7934656.

Change-Id: I57117bff3bc97564d5050e4890d56b3f159aae18
---
 bluetoothd.te       |  8 +++-----
 bluetoothd_snoop.te | 10 +++-------
 2 files changed, 6 insertions(+), 12 deletions(-)

diff --git a/bluetoothd.te b/bluetoothd.te
index 72c79fc..a572e37 100644
--- a/bluetoothd.te
+++ b/bluetoothd.te
@@ -2,14 +2,12 @@ type bluetoothd, domain;
 type bluetoothd_exec, exec_type, file_type;
 type bluetoothd_main_exec, exec_type, file_type;
 
+# FIXME: this is still here because of logwrapper
+permissive bluetoothd;
+
 # Start bluetoothd from init
 init_daemon_domain(bluetoothd)
 
-# logwrapper domain transition
-domain_auto_trans(logwrapper, bluetoothd_main_exec, bluetoothd)
-domain_auto_trans(logwrapper, bluetoothd_exec, bluetoothd)
-allow bluetoothd logwrapper:fd use;
-
 # Data file accesses
 allow bluetoothd bluetooth_data_file:dir w_dir_perms;
 allow bluetoothd bluetooth_data_file:notdevfile_class_set create_file_perms;
diff --git a/bluetoothd_snoop.te b/bluetoothd_snoop.te
index e082341..626b583 100644
--- a/bluetoothd_snoop.te
+++ b/bluetoothd_snoop.te
@@ -1,16 +1,12 @@
 type bluetoothd_snoop, domain;
 type bluetoothd_snoop_exec, exec_type, file_type;
 
+# FIXME: this is still here because of logwrapper
+permissive bluetoothd;
+
 # Start bluetoothd_snoop from init
 init_daemon_domain(bluetoothd_snoop)
 
-# logwrapper domain transition
-domain_auto_trans(logwrapper, bluetoothd_snoop_exec, bluetoothd_snoop)
-allow bluetoothd_snoop logwrapper:fd use;
-
-# needs /system/bin/log access
-allow bluetoothd_snoop devpts:chr_file rw_file_perms;
-
 # directory search and read caps
 allow bluetoothd_snoop self:capability dac_read_search;
 # use raw and packet sockets caps
-- 
2.9.3

